docs/guides/code-style-guide.md

```diff
@@ -0,0 +1,160 @@
+---
+id: code-style-guide
+title: Code Style Guide
+sidebar_label: Code Style Guide
+---
+
+## Why should I read this?
+
+There are many different ways of writing code that achieve the same goal. However, programmers generally agree that there are some ways of writing code that are more understandable and *readable* than other ways.
+
+To help write more understandable code, programmers follow a *style guide*. It provides guidelines for how to write code, how to structure it, common problems to avoid, etc.
+
+## Using Prettier to format code automatically
+
+Some of the guidelines are so common that there are automatic tools to do it for you! One of these tools is called Prettier. You should have installed this tool when you [set up your VS Code extensions](https://docs.codeyourfuture.io/course-processes/before-the-course/getting-setup#extensions).
+
+Prettier is a tool that rearranges your code (called *formatting*). It follows a set of rules that programmers have agreed make your code easier to read and understand.
+
+Some common problems you may see when running Prettier:
+
+- If you write some code that is invalid, then running Prettier will not work and your code won't be formatted. If this happens, then you will see some red "squiggly" lines under under your code. A common problem is a missing bracket, so try looking for that first
```

Comment:
Maybe an image would be better here rather than trying to describe it?

Reply:
Done in e55c921.